edge-apps/google-calendar/screenly.yml

```diff
@@ -0,0 +1,75 @@
+---
+syntax: manifest_v1
+id: 01JN23EC946NKCSYF5D64V4D53
+description: Screenly Google Calendar Edge App
+icon: https://playground.srly.io/edge-apps/google-calendar/static/img/icon.svg
+author: Screenly, Inc.
+ready_signal: true
+settings:
+  bypass_cors:
+    type: string
+    title: Bypass CORS
+    optional: true
+    help_text: Bypass CORS for the iCal URL
+    default_value: 'false'
+  calendar_mode:
+    type: string
+    title: Calendar Mode
+    optional: true
+    help_text: Choose between monthly, weekly, or daily calendar view
+    default_value: 'monthly'
+  enable_analytics:
+    type: string
+    default_value: 'true'
+    title: Enable Analytics
+    optional: true
+    help_text: Whether to disable Sentry and Google Analytics integrations
+    is_global: true
+  ical_url:
+    type: secret
+    title: iCal URL
+    optional: false
+    help_text: Specify the URL to the iCal feed
+  tag_manager_id:
+    type: string
+    default_value: GTM-P98SPZ9Z
+    title: Google Tag Manager ID
+    optional: true
+    help_text: Specify a Google Tag Manager ID
+    is_global: true
+  theme_primary_color:
+    type: string
+    title: Theme Primary Color
+    optional: true
+    help_text: Override the primary theme color (in hex format, e.g. #7E2CD2)
+    default_value: '#7E2CD2'
+  theme_secondary_color:
+    type: string
+    title: Theme Secondary Color
+    optional: true
+    help_text: Override the secondary theme color (in hex format, e.g. #C9CDD0)
+    default_value: '#C9CDD0'
```

Comment:
These colours should be fetched from brand settings, and they will change as the user changes the values in the "branding-customization" property in their Screenly account.  @vpetersson Do you think we should mention this explicitly? (In Other apps, only the "brand settings" can alter these values)

Reply:
also as per theme - the "theme_tertiary" ,  "theme_accent" and "theme_text_color" are fixed.  Can you check that Adnan. 

ping @nicomiguelino

Reply:
@salmanfarisvp, how about dark mode?

Reply:
Our earlier app had an extra **setting** labelled "theme," which defaults to "light." Users can select either "light" or "dark". Based on their choice, the system will use either:

- screenly_color_light
- screenly_logo_light  
or  
- screenly_color_dark
- screenly_logo_dark

what do you think about this method?

Reply:
https://developer.screenly.io/edge-apps/#global-branding-settings